<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cửa sổ chính</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tải tệp tin</string>
     </attribute>
     <widget class="QPushButton" name="tai_xuong">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tải xuống</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="duong_dan_tai">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Nhập đường dẫn cần tải</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="duong_dan_luu">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Chọn nơi lưu tệp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chon_noi_luu">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>150</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/icons/duyettep.png</normaloff>:/icons/duyettep.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QProgressBar" name="thanh_tien_trinh">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tải video từ Youtube</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tải video đơn</string>
       </attribute>
       <widget class="QLineEdit" name="duong_dan_video_don">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>521</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nhập đường dẫn video</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="noi_luu_video_don">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>521</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Chọn đường dẫn lưu tệp</string>
        </property>
       </widget>
       <widget class="QPushButton" name="duyet_tep_video_don">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>110</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icons/duyettep.png</normaloff>:/icons/duyettep.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
       <widget class="QProgressBar" name="tien_trinh_video_don">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>200</y>
          <width>581</width>
          <height>21</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="tai_xuong_video_don">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>250</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Tải xuống</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>160</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chất lượng video</string>
        </property>
       </widget>
       <widget class="QComboBox" name="chat_luong_video_don">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>160</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="kiem_tra_video_don">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>50</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icons/check.svg</normaloff>:/icons/check.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>65</width>
          <height>65</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="thoi_gian_video_don">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>346</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tải theo danh sách phát</string>
       </attribute>
       <widget class="QLineEdit" name="duong_dan_danh_sach">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>541</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nhập đường dẫn danh sách</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="duong_dan_luu_danh_sach">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>541</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Chọn nơi lưu video</string>
        </property>
       </widget>
       <widget class="QPushButton" name="duyet_tep_danh_sach">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>80</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icons/duyettep.png</normaloff>:/icons/duyettep.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>140</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chất lượng video</string>
        </property>
       </widget>
       <widget class="QComboBox" name="chat_luong_danh_sach">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>130</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="tien_trinh_tai_danh_sach">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>611</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="tai_xuong_danh_sach">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>300</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Bắt đầu tải</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Video hiện tại</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="video_hien_tai">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>200</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Danh sách video</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="tong_so_video">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>190</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="kiem_tra_danh_sach">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>30</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icons/taixuong.svg</normaloff>:/icons/taixuong.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="lay_danh_sach_phat">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
